### DISCUSSION TITLE
LiquidCrystalIO on 2nd i2c bus

### DISCUSSION THREAD

**Discussion by tomelgato** [davetcc/IoAbstraction#192] [Q&A] [answered]
As the title says, how can i connect my LCD on the 2nd I2C bus (wire1)?

**Comment by davetcc** [comment, marked as answer]
If you're using Ioabstraction stand-alone, just use the constructor that allows you to provide a different wire implementation:

https://github.com/davetcc/LiquidCrystalIO/blob/master/src/LiquidCrystalIO.h#L88

IE that would probably be:

     LiquidCrystal(uint8_t rs, uint8_t rw, uint8_t en, uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3,
                  LiquidCrystal::BackLightPinMode mode, IoAbstractionRef ioMethod);

Otherwise, if using with tcMenu it would be as simple as:

![image](https://github.com/davetcc/IoAbstraction/assets/12195465/20ecc58b-a67c-4d27-aa43-f2506a268052)

**Reply by davetcc** [reply]
These two definitions at the bottom of the LCD header file should give you some idea how that may look:

https://github.com/davetcc/LiquidCrystalIO/blob/master/src/LiquidCrystalIO.h#L395

**Reply by tomelgato** [reply]
ah cool. did completely miss the option in tcmenu. thanks!
